### PR TITLE
SC-12289 Multiple Device Types documentation

### PR DESCRIPTION
**Pull request by hsalsem**
Added information regarding what changing the device type does, as well as information on the new option to create multiple monitors in the same flow. Also added instructions on how to see if we support a certain device to the FAQ.

### CODE REVIEW

File: docs/synthetics/browser-monitor.md [hunk 1]
```diff
@@ -29,7 +32,7 @@ By default, the UI adds the below conditions while creating a browser monitor. Y
 
 ## Screenshots
 
-The browser monitor script allows the collection of page screenshots at any point during the execution. This can be collected using [page.screenshot()](https://github.com/puppeteer/puppeteer/blob/master/docs/api.md#pagescreenshotoptions) Puppeteer API. JPEG and PNG image types are supported. Currently, the number of screenshots per run is limited to one. On failure of the script due to errors like navigation timeout, assertion failed, etc., a screenshot `error.png` will be collected for analysis.
+The browser monitor script allows the collection of page screenshots at any point during the execution. This can be collected using [page.screenshot()](https://github.com/puppeteer/puppeteer/blob/master/docs/api.md#pagescreenshotoptions) Puppeteer API. The resolution of a screenshot corresponds to the selected `Device type` (`Desktop` defaults to `1920x1080`). JPEG and PNG image types are supported. The number of screenshots per run is limited to three for the Pro plan, and to one for the Pay-As-You-Go and Standard plans. On failure of the script due to errors like navigation timeout, assertion failed, etc., a screenshot `error.png` will be collected for analysis.
```
Comment (otisg) [inline docs/synthetics/browser-monitor.md:35]: May want to generalize the wording around number of screenshots and link to /pricing#synthetics in order to stick with DRY and avoid having this out of sync if things change.

File: docs/synthetics/faq.md [hunk 2]
```diff
@@ -81,6 +81,9 @@ Frankfurt (eu-central-1)   - 18.159.198.132 <br/>
 Ireland (eu-west-1)        - 54.77.99.140   <br/>
 Sao Paulo (sa-east-1)      - 54.207.1.35    <br/>
 
+### Where can I see if a certain device is supported for browser monitors?
+You can simply type the name of your desired device into the _Device type_ dropdown while creating a monitor and see if a matching option will be displayed. You can also find the list of devices we support, as well as their specific properties, in the official [Puppeteer docs](https://github.com/puppeteer/puppeteer/blob/v13.6.0/src/common/DeviceDescriptors.ts).
```
Comment (otisg) [inline docs/synthetics/faq.md:85]: I assume you used a specific version here on purpose.

**Review by otisg — APPROVED** [review]
Minor comments.  LGTM.

File: docs/synthetics/faq.md (hunk 2 above)
Comment (hsalsem) [reply docs/synthetics/faq.md:85]: Yes, those are the devices we're currently using

**Pull request merged**